app/templates/events/view/tickets/ticket-details.hbs

```diff
@@ -0,0 +1,111 @@
+<div class="ui segment">
+  <div class="twelve wide column">
+    <div class="ui statistics">
+      <div class="statistic"><div class="value">4343</div><div class="label">{{t 'Orders'}}</div></div>
+      <div class="statistic"><div class="value">0.00</div><div class="label">{{t 'Sales(US $)'}}</div></div>
+      <div class="statistic"><div class="value">1</div><div class="label">{{t 'Tickets sold'}}</div></div>
+    </div>
+  </div>
+  <br>
+  <div class="row">
+    <h4>Order Summary&nbsp;<span>By order status</span></h4>
```

Comment:
Use semantic header for this ... and put 'By order status' as the subheader there